frontend/src/lib/i18n/locales/kr.json

```diff
@@ -0,0 +1,649 @@
+    },
+    "bulk_add_members": {
+      "add_button": "멤버 대량 추가/생성",
+      "explanation": "입력된 모든 로그인 및 이메일을 이 프로젝트에 추가합니다. 계정이 없는 사용자의 새 계정이 자동으로 생성됩니다.",
+      "modal_title": "멤버 대량 추가/생성",
+      "submit_button": "멤버 추가/생성",
+      "finish_button": "닫기",
+      "shared_password": "공유 비밀번호",
+      "shared_password_description": "새 사용자용",
+      "usernames": "로그인 또는 이메일 (줄마다 하나씩)",
+      "usernames_description": "이것은 새 사용자의 **송수신 로그인**이 됩니다",
+      "invalid_username": "잘못된 로그인/사용자 이름: {username}. 문자, 숫자 및 밑줄(_) 문자만 허용됩니다.",
+      "empty_user_field": "이메일 주소 및/또는 로그인을 입력하십시오",
+      "creator_must_have_email": "프로젝트를 생성하려면 이메일 주소가 있어야 합니다.",
+      "members_added": "{addedCount}명의 새 {addedCount, plural, one {멤버가} other {멤버들이}} 프로젝트에 추가되었습니다.",
+      "existing_added_members": "{existedCount}명의 기존 {existedCount, plural, one {사용자가} other {사용자들이}} 이미 존재했습니다.",
+      "already_members": "{count, plural, one {#명의 사용자가} other {#명의 사용자들이}} 이미 프로젝트에 있습니다.",
+      "accounts_created": "{createdCount}개의 새 {createdCount, plural, one {계정이} other {계정들이}} 제공된 비밀번호로 생성되었습니다.",
+      "accounts_invited": "{invitedCount}명의 새 {invitedCount, plural, one {사용자가} other {사용자들이}} 이메일로 초대되었습니다.",
+      "invalid_email_address": "잘못된 이메일 주소: {email}.",
+    },
+    "change_role_modal": {
+      "title": "{name}의 역할 선택"
+    },
+    "reset_project_modal": {
+      "title": "프로젝트 {code} 재설정",
+      "submit": "프로젝트 재설정",
+      "close": "닫기",
+      "i_have_working_backup": "작동하는 백업이 있습니다",
+      "leave_project_empty": "프로젝트를 비워 둡니다",
+      "next": "다음",
+      "back": "뒤로",
+      "download_instruction": "먼저, 프로젝트의 백업을 다운로드하여 3단계에서 복원에 사용할 수 있도록 합니다:",
+      "download_button": "프로젝트 백업 다운로드",
+      "confirm_downloaded": "프로젝트 백업을 다운로드하고 작동하는지 확인했음을 확인합니다. 프로젝트 리포지토리 내용을 완전히 재설정/삭제할 준비가 되었습니다.",
+      "confirm_downloaded_error": "백업을 다운로드했음을 확인하십시오",
+      "confirm_project_code": "재설정을 확인하려면 프로젝트 코드를 입력하십시오",
+      "confirm_project_code_error": "재설정을 확인하려면 프로젝트 코드를 입력하십시오",
+      "reset_project_notification": "프로젝트 {code}이(가) 성공적으로 재설정되었습니다.",
+      "upload_instruction": "프로젝트 리포지토리가 성공적으로 재설정되었습니다. 이제 복원을 위해 zip 파일을 업로드하십시오:",
+      "upload_project": "프로젝트 업로드",
+      "select_zip": "프로젝트 zip 파일",
+      "should_only_contain_hg": "루트에 .hg 폴더만 포함되어야 합니다",
+      "reset_success": "프로젝트가 성공적으로 재설정되었습니다.",
+      "backup_step": "백업",
+      "reset_step": "재설정",
+      "restore_step": "복원",
+      "finished_step": "완료",
+      "reset_in_progress": "프로젝트 재설정 진행 중",
+      "click_to_continue": "계속하려면 클릭"
+    },
+    "notifications": {
+      "role_change": "{name}의 프로젝트 역할이 {role}로 설정되었습니다.",
+      "reset_project": "프로젝트 {code}이(가) 재설정되었습니다.",
+      "user_delete": "{name}이(가) 제거되었습니다.",
+      "rename_project": "프로젝트 이름이 {name}(으)로 설정되었습니다.",
+      "promote_project": "프로젝트가 초안 상태에서 실제 프로젝트로 변경되었습니다.",
+      "describe": "프로젝트 설명이 업데이트되었습니다.",
+      "add_member": "{email}이(가) 프로젝트에 추가되었습니다.",
+      "member_invited": "{email}에게 프로젝트 초대 이메일이 발송되었습니다."
```

Comment:
Here, too, I suspect there should be spaces between the substitutions and the rest of the words.